src/hackerspace_online/urls.py

```
@@ -40,8 +40,10 @@
     re_path(r'^$', views.home, name='home'),
     re_path(r'^a/simple/life/is/its/own/reward/', views.simple, name='simple'),
     # quest_manager
-    re_path(r'^library/', include('library.urls', namespace='library')),
-    re_path(r'^quests/', include('quest_manager.urls', namespace='quests')),
+    url(r'^library/', include('library.urls', namespace='library')),
```

Ruff (0.8.2)

Undefined name `url`

(F821)